### PR TITLE
Add support for starting stopping and refreshing pools

### PR DESCRIPTION
**Pull request by mulkieran**
Closes https://github.com/stratis-storage/stratis-cli/issues/725
Related stratis-storage/project#296
Related https://github.com/stratis-storage/stratis-cli/issues/866

### CODE REVIEW

**Comment by mulkieran** [comment]
coverage task will fail.

**Comment by mulkieran** [comment]
stratisd dependent tests should all fail.

**Comment by mulkieran** [comment]
rebased.

**Comment by mulkieran** [comment]
The bug in listing stopped pools should be fixed now.

**Review by drckeefe — APPROVED** [review]

**Comment by mulkieran** [comment]
rebased.

File: src/stratis_cli/_actions/_pool.py [hunk 1]
```diff
@@ -468,31 +468,41 @@ def list_stopped_pools(namespace):
 
         stopped_pools = _fetch_stopped_pools_property(proxy)
 
-        def na_string(value):
+        def maybe_inconsistent(value):
             """
-            Get a cell value or N/A if None.
+            Take a value that represents possibly inconsistency via option type.
```
Comment (bmr-cymru) [inline src/stratis_cli/_actions/_pool.py]: ^possibly^possible?

**Review by bmr-cymru — APPROVED** [review]
One possible typo in a comment but looks good otherwise

**Comment by mulkieran** [comment]
This passes all tests again https://github.com/stratis-storage/stratisd/pull/2949 .

**Comment by mulkieran** [comment]
Should we allow ```capture_key``` in the "pool start" subcommand.

**Comment by mulkieran** [comment]
All tests and coverage succeed when run agains: https://github.com/stratis-storage/stratisd/pull/2949 .

**Pull request merged**